var/spack/repos/builtin/packages/mesa/package.py

```diff
@@ -92,6 +92,8 @@ class Mesa(MesonPackage):
     # OpenGL ES requires OpenGL
     conflicts('~opengl +opengles')
 
+    patch('https://cgit.freedesktop.org/mesa/mesa/patch/meson.build?id=054dd668a69acc70d47c73abe4646e96a1f23577', sha256='36096a178070e40217945e12d542dfe80016cb897284a01114d616656c577d73', when='@21.0.0:21.0.3')
```

Comment:
Can you add a brief comment explaining what the patch does?

Reply:
good idea, just pushed a commit with a comment